package.json

```diff
@@ -17,6 +17,8 @@
     ]
   },
   "dependencies": {
+    "@react-navigation/bottom-tabs": "^6.5.2",
+    "@react-navigation/native": "^6.1.1",
     "expo": "~44.0.0",
```

Comment:
we probably want to migrate our apps to the newest version of Expo to avoid the Expo Go App compatibility issue

Reply:
oh right good catch